plugins/modules/packaging/os/pkgng.py

```diff
@@ -90,6 +90,13 @@
         required: false
         type: bool
         default: no
+    ignoreosver:
```

Comment:
```suggestion
    ignore_osver:
```
Makes it somewhat more readable.